scheduler/src/cook/mesos/util.clj

```diff
@@ -28,6 +28,49 @@
           db)
        (map first)))
 
+(defn job-ent->container
+  "Take a job entity and return its container"
+  [db job job-ent]
+  (clojure.pprint/pprint "=job-ent->container=")
+  (if (contains? job-ent :job/container)
+    (let [ceid (:db/id (:job/container job-ent))
+          cmap (d/pull db "[*]" ceid)
+          rm-dbids (fn rm-dbids [m]
+                     (cond
+                       (map? m)
+                       (let [sm (filter (fn [p]
+                                          (not(= :db/id (first p)))) m)]
+                         (into {} (map (fn [p]
+                                         [(first p) (rm-dbids (second p))])
+                                       sm)))
+                       (vector? m)
+                       (mapv rm-dbids m)
+                       :else
+                       m))
+          fixup-keywords (fn fixup-keywords [m]
+                           (cond
+                             (map? m)
+                             (into {} (map (fn [p]
+                                             (let [k (first p)
+                                                   v (second p)]
+                                               [ (if (keyword? k)
+                                                   (keyword(name k)) k)
```

Comment:
we typically prefer to explicitly list the from and to keywords--although the dynamic casts are shorthand for now, they're harder to understand 6 months down the line.